jbpm-installer/src/main/resources/db/ddl-scripts/db2/quartz_tables_db2.sql

```diff
@@ -69,8 +69,8 @@ CREATE TABLE qrtz_simprop_triggers
     LONG_PROP_2 BIGINT,
     DEC_PROP_1 NUMERIC(13,4),
     DEC_PROP_2 NUMERIC(13,4),
-    BOOL_PROP_1 VARCHAR(1),
-    BOOL_PROP_2 VARCHAR(1),
+    BOOL_PROP_1 INTEGER,
+    BOOL_PROP_2 INTEGER,
```

Comment:
In the quartz db2 schemas (tables_db2 and tables_db2_v72) boolean are mapped always to varchar(1). This is different for schemas (tables_db2_v8, tables_db2_v95), where SQL boolean type is mapped into:

- varchar(1) for BOOL_PROP_1, BOOL_PROP_2 at qrtz_simprop_triggers table
- integer for is_durable, is_nonconcurrent, is_update_data, requests_recovery at qrtz_job_details table and is_nonconcurrent, requests_recovery at qrtz_fired_triggers table.

Now I've unified like in tables_db2, but if you don't feel comfortable because this change is not backward-compatible, I can leave it as it was (db2 is already failing for upgrading scripts)